src/panel/app_panel.js

```diff
@@ -281,6 +290,19 @@ export default class AppPanel {
     });
   }
 
+  openEvents(params) {
+    const { type: eventName, q: query, ...otherOptions } = params;
```

Comment:
The `q: query` part can be removed here and everywhere else (props especially), as the API doesn't support (or at least it's not a parameter used by `poiEventLoad`).